README.md

```diff
@@ -1,14 +1,32 @@
-# ImagePicker
+# ImagePicker :camera:
 
-[![CI Status](http://img.shields.io/travis/hyperoslo/ImagePicker.svg?style=flat)](https://travis-ci.org/hyperoslo/ImagePicker)
 [![Version](https://img.shields.io/cocoapods/v/ImagePicker.svg?style=flat)](http://cocoadocs.org/docsets/ImagePicker)
 [![License](https://img.shields.io/cocoapods/l/ImagePicker.svg?style=flat)](http://cocoadocs.org/docsets/ImagePicker)
 [![Platform](https://img.shields.io/cocoapods/p/ImagePicker.svg?style=flat)](http://cocoadocs.org/docsets/ImagePicker)
 
+## Description
+
+![ImagePicker](https://github.com/hyperoslo/ImagePicker/master/Resources/ImagePickerPresentation.png)
+
+**ImagePicker** is an all-in-one camera solution for your iOS app. It let's your users to select images from the library and take pictures at the same time. As a developer you get notified of all the user interactions and get the beautiful UI for free, out of the box, it's just that simple.
```

Comment:
It let's your users **to** select

I think **to** can be removed here :sunglasses: